packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -554,10 +571,11 @@ class VirtualListBasic extends Component {
 
 	getXY = (primaryPosition, secondaryPosition) => (this.isPrimaryDirectionVertical ? {x: secondaryPosition, y: primaryPosition} : {x: primaryPosition, y: secondaryPosition})
 
-	getClientSize = (node) => ({
-		clientWidth: node.clientWidth,
-		clientHeight: node.clientHeight
-	})
+	getClientSize = (props) => {
+		const clientSize = props.clientSize || props.scrollContentRef.current;
```

Comment:
I'm not sure that it is the right way for app developers to pass the `clientSize` prop to the sandstone/VirtualList.

It's right to do it in moonstone/VirtualList because the `width` and the `height` are same with the `clientWidth` and the `clientHeight` if no scrollbars. But for sandstone, it is hard for app developers to calculate the width, height of the sandstone/VirtualList because they  have to add proper some size of padding and margin for fadeout area and scrollbar area.

For now, I think that it would be better to pass not `clientSize` but `width` and `height` to the sandstone/VirtualList. Please let me know your thought.

Reply:
I'm sorry but I'm very confused. Is your point about `prop` for app developers or additional post-processing for each themed library?

Reply:
Note: @ybsung pointed out the prop `clientSize` may not be comfortable to our app devs on other themed libraries as I discussed off-line. This topic is valid but we don't want to discuss it here since it is a topic of API. So, we will discuss it separately if needed.